presto-hive/src/main/java/com/facebook/presto/hive/HiveConnectorFactory.java

```diff
@@ -124,6 +129,7 @@ public Connector create(String connectorId, Map<String, String> config, Connecto
             HiveSessionProperties hiveSessionProperties = injector.getInstance(HiveSessionProperties.class);
             HiveTableProperties hiveTableProperties = injector.getInstance(HiveTableProperties.class);
             ConnectorAccessControl accessControl = new PartitionsAwareAccessControl(injector.getInstance(ConnectorAccessControl.class));
+            Set<Procedure> procedures = injector.getInstance(Key.get(new TypeLiteral<Set<Procedure>>(){}));
```

Comment:
Please run the code formatter. I think there is supposed to be a space here: `() {}`

(search the code base for other usages of `TypeLiteral` to confirm)